### PR TITLE
Improve doc generation

### PR DESCRIPTION
**Pull request by autra**
Add the api doc and display version.

Probably a bit of noise in generated api doc though. And a lot of doc to complete.

### CODE REVIEW

**Comment by autra** [comment]
@delhomer WDYT?

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Doc of py3dtiles
+
+## How to generate the doc?
+
+API doc structure is generated with 
+```
+SPHINX_APIDOC_OPTIONS sphinx-apidoc -o ./api ../py3dtiles
+```
+This command needs to be used only when new files are added (TODO check if shpinx-autoapi wouldn't do this job for us?)
```
Comment (delhomer) [inline docs/README.md]: `shpinx-autoapi` -> `sphinx-autoapi`

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Doc of py3dtiles
+
+## How to generate the doc?
+
+API doc structure is generated with 
+```
+SPHINX_APIDOC_OPTIONS sphinx-apidoc -o ./api ../py3dtiles
```
Comment (delhomer) [inline docs/README.md]: I get a "SPHINX_APIDOC_OPTIONS : command not found" message.

**Review by delhomer — COMMENTED** [review]
@autra Nice job!

A lot of method docstrings are indeed missing, however the doc auto-generation seems to work fine.

Just a question about a command you specify, and that I do not understand.

File: docs/README.md (hunk 2 above)
Comment (autra) [reply docs/README.md]: Yeah I forgot a bit apparently

Comment (autra) [reply docs/README.md]: I updated this part @delhomer

**Comment by autra** [comment]
> A lot of method docstrings are indeed missing, however the doc auto-generation seems to work fine.

Do you think we should remove docs that are not completed? Or is it better this way, as it shows the work we still to do?

**Comment by delhomer** [comment]
> > A lot of method docstrings are indeed missing, however the doc auto-generation seems to work fine.
> 
> Do you think we should remove docs that are not completed? Or is it better this way, as it shows the work we still to do?

In my opinion, that's OK to let it as it is! The message sounds like "contributions are welcome", in a certain point of view.

File: docs/README.md (hunk 2 above)
Comment (delhomer) [reply docs/README.md]: Ok it sounds fine. Maybe we could specify as a prerequisite the following command:
```
pip install -e .[doc]
```
Pretty implicit for pythonistas, however I prefer to make it explicit.

Comment (autra) [reply docs/README.md]: Yep. Done.

**Pull request merged**